### ISSUE TITLE
Failed search results message needs to give tips for trying another search

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2726]
As per Trusted Tester, the search results page should give more instruction on what to do with a failed search. Ours only says the search failed - it doesn't give suggestiong for fixing the error:

![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/1d64388d-25a9-4039-b925-2186ba4f28f9)

---
---

We can copy the one on onrr.gov or write something similar but a note add to check spelling:
![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/53611004-b22a-4899-8889-40b2f4a6c603)
